### ISSUE TITLE
Region rule complains on html element

### ISSUE THREAD

**Issue by badtant** [dequelabs/axe-core#215]
When i enable the region rule I get this message:
```
failureSummary: "Fix any of the following:  Content not contained by an ARIA landmark"
html: "<html lang="sv">"
impact: "moderate"
```

No quite sure what I should fix. Or is this a false error?

**Comment by marcysutton** [comment]
I can repro it and it fails even when the content is wrapped in a landmark region. This one definitely needs to be fixed before we enable it by default.

**Comment by dylanb** [comment]
ok, the way this rule works is that it looks to make sure that each portion of the page that contains content is also contained in a landmark role. If this is not the case, it returns the HTML document as the violating node and the list of nodes outside a landmark as the related nodes.

@marcysutton test a page like this

```
<html lang="sv">
<head>
<title>some title</title>
</head>
</body>
<div role="main">
content
</div>
</body>
</html>
```

**Comment by badtant** [comment]
Cant see where I find related nodes? @dylanb

**Comment by dylanb** [comment]
If you look in the `nodes` array for each rule, you will see a `relatedNodes` array with the information. Also, if you look at this example implementation, you can see how to turn the results object into HTML https://github.com/dequelabs/axe-core/blob/master/doc/examples/html-handlebars.md

**Comment by badtant** [comment]
Ahh, I had to use v2

**Comment by dylanb** [comment]
Can I close this issue?

**Comment by badtant** [comment]
Yes!

**Comment by marcysutton** [comment]
I found out a little more about this one: the region rule requires `role="banner"` or `role="main"` to be bolted on to HTML5 elements, arguably to gain support from IE which did not expose the implicit roles.

**Comment by badtant** [comment]
One thing that I'm wondering about is for instance a "skip to content"-link. Should that be placed inside the header (role banner)? Otherwise I'll get the error but I don't think it belongs in there...

**Comment by dylanb** [comment]
if the link is an internal link, it will be ignored by the rule
